### PR TITLE
Add some decoration to tool CLI

### PR DESCRIPTION
**Pull request by charliermarsh**
Mostly small things. I added entrypoint counts and bolded the executable names.

Closes https://github.com/astral-sh/uv/issues/4815.

### CODE REVIEW

File: crates/uv/src/commands/tool/uninstall.rs [hunk 1]
```diff
@@ -67,12 +68,14 @@ pub(crate) async fn uninstall(
         }
     }
 
+    let s = if entrypoints.len() == 1 { "" } else { "s" };
     writeln!(
         printer.stderr(),
-        "Uninstalled: {}",
+        "Uninstalled {} executable{s}: {}",
```
Comment (charliermarsh) [inline crates/uv/src/commands/tool/uninstall.rs:74]: Are you opposed to using "executable" for user-facing copy?

**Comment by charliermarsh** [comment]
(I will update fixtures once approved, just a bit tedious.)

File: crates/uv/src/commands/tool/uninstall.rs (hunk 1 above)
Comment (zanieb) [reply crates/uv/src/commands/tool/uninstall.rs:74]: It feels a little weird, I don't have any great alternatives though. "tool entry points"? eh.

File: crates/uv/src/commands/tool/install.rs [hunk 2]
```diff
@@ -265,7 +266,7 @@ pub(crate) async fn install(
         .context("Failed to create executable directory")?;
 
     debug!(
-        "Installing tool entry points into {}",
+        "Installing tool entrypoints into: {}",
```
Comment (zanieb) [inline crates/uv/src/commands/tool/install.rs:269]: These are always two words in the official documentation

https://packaging.python.org/en/latest/specifications/entry-points/

Comment (zanieb) [reply crates/uv/src/commands/tool/install.rs:269]: (I like how entrypoints looks too, but I think we should be consistent. I've been following the official docs here)

File: crates/uv/src/commands/tool/uninstall.rs (hunk 1 above)
Comment (zanieb) [reply crates/uv/src/commands/tool/uninstall.rs:74]: We use tool entry points everywhere else in this output, right?

File: crates/uv/src/commands/tool/install.rs (hunk 2 above)
Comment (charliermarsh) [reply crates/uv/src/commands/tool/install.rs:269]: Oh strange, thank you, I can change it.

File: crates/uv/src/commands/tool/uninstall.rs (hunk 1 above)
Comment (charliermarsh) [reply crates/uv/src/commands/tool/uninstall.rs:74]: But entry point is not as general... We also support data scripts which aren't entry points. I feel like entry point is sort of an implementation detail.

Comment (zanieb) [reply crates/uv/src/commands/tool/uninstall.rs:74]: Fair. Why not just "commands" then?

Comment (zanieb) [reply crates/uv/src/commands/tool/uninstall.rs:74]: I don't have strong feelings here.

Comment (zanieb) [reply crates/uv/src/commands/tool/uninstall.rs:74]: Cargo uses `executable`, e.g.

```
   Replacing /Users/zb/.cargo/bin/cargo-nextest
    Replaced package `cargo-nextest v0.9.70` with `cargo-nextest v0.9.72` (executable `cargo-nextest`)
```

Comment (charliermarsh) [reply crates/uv/src/commands/tool/uninstall.rs:74]: Ok, I'm gonna run with executable. I can change other user-facing copy too for consistency.

**Pull request merged**